src/app/app-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { FormComponent } from './components/form/form.component';
+import { HomeComponent } from './components/home/home.component';
+export { Routes } from '@angular/router';
```

Comment:
Try to use some kind of plugin to auto sort imports when you are savings a file.